os.bpmsuite.executionserver/added/launch/bpmsuite-executionserver.sh

```diff
@@ -214,13 +207,12 @@ function configure_controller_access {
         JBOSS_BPMSUITE_ARGS="${JBOSS_BPMSUITE_ARGS} -Dorg.kie.server.controller=${kieServerControllerUrl}"
     fi
     # user/pwd
-    local kieServerControllerUser=$(find_env "KIE_SERVER_CONTROLLER_USER" "controllerUser")
-    local kieServerControllerPwd=$(find_env "KIE_SERVER_CONTROLLER_PWD" "controller1!")
-    JBOSS_BPMSUITE_ARGS="${JBOSS_BPMSUITE_ARGS} -Dorg.kie.server.controller.user=${kieServerControllerUser}"
-    JBOSS_BPMSUITE_ARGS="${JBOSS_BPMSUITE_ARGS} -Dorg.kie.server.controller.pwd=${kieServerControllerPwd}"
+    JBOSS_BPMSUITE_ARGS="${JBOSS_BPMSUITE_ARGS} -Dorg.kie.server.controller.user=$(get_kie_server_controller_user)"
+    JBOSS_BPMSUITE_ARGS="${JBOSS_BPMSUITE_ARGS} -Dorg.kie.server.controller.pwd=$(get_kie_server_controller_pwd)"
     # token
-    if [ "${KIE_SERVER_CONTROLLER_TOKEN}" != "" ]; then
-        JBOSS_BPMSUITE_ARGS="${JBOSS_BPMSUITE_ARGS} -Dorg.kie.server.controller.token=${KIE_SERVER_CONTROLLER_TOKEN}"
+    local kieServerControllerToken="$(get_kie_server_controller_token)"
```

Comment:
is it a function? if so, I didn't find it in the changes.

Reply:
Look in bpmsuite-security.sh.

Reply:
I didn't find because there is a typo in the function, there is "get_kie_server_conroller_token"